app/admin/job_posting.rb

```diff
@@ -4,9 +4,46 @@
     :description,
     :how_to_apply,
     :full_time,
-    :company
+    :company,
+    :akismet_spam,
+    :user_ip,
+    :user_agent,
+    :referrer
   )
 
+  index do
+    id_column
+    column :company
+    column :full_time
+    column :title
+    column :description
+    column :created_at
+    column 'Is spam?', :akismet_spam
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.